pkg/webui/console/containers/lora-cloud-das-form/index.js

```diff
@@ -95,6 +95,7 @@ const defaultValues = {
   data: {
     token: '',
     server_url: LORA_CLOUD_DAS.DEFAULT_SERVER_URL,
+    f_port_set: LORA_CLOUD_DAS.DEFAULT_PORT,
```

Comment:
This is a single port, while we should provide multiple of them per issue description.